x-pack/plugins/reporting/server/lib/store/report.ts

```diff
@@ -4,94 +4,109 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import moment from 'moment';
 // @ts-ignore no module definition
 import Puid from 'puid';
 import { JobStatuses } from '../../../constants';
-import { LayoutInstance } from '../layouts';
+import { LayoutParams } from '../layouts';
+import { TaskRunResult } from '../tasks';
 
-/*
- * The document created by Reporting to store in the .reporting index
- */
-interface ReportingDocument {
-  _id: string;
-  _index: string;
-  _seq_no: unknown;
-  _primary_term: unknown;
+export interface ReportSource {
   jobtype: string;
   created_by: string | false;
   payload: {
-    headers: string; // encrypted headers
+    browserTimezone: string;
+    headers: string;
+    layout?: LayoutParams;
     objectType: string;
-    layout?: LayoutInstance;
```

Comment:
this fixes a bug: LayoutInstance is a class, not an interface. Not sure how it ended up this way, but it should have always been `LayoutParams`